src/app/new-page/component/card/card.component.html

```diff
@@ -0,0 +1,20 @@
+<div
+  attr.id="card-{{ cardIndex }}"
+  class="cardContent"
+  *ngIf="ready"
+  [hidden]="card.isHidden"
+  [ngClass]="{
+    hasHeading: !(card.listeners.length && card.dismissListeners.length)
+  }"
+>
+  <h4
+    *ngIf="!(card.listeners.length && card.dismissListeners.length)"
```

Comment:
hmm, I'm curious why we are hiding the card header if there are `listeners` or `dismissListeners` defined

Reply:
@dr-bizz I'm assuming this is the same behavior that the previous renderer was doing. In which case we don't need to address this with the PR, but it's something we should look into after the fact

Reply:
@frett Yes, it was. I couldn't find a place where the card had Listeners and dismissListeners to see why it wouldn't show the header.
I've removed the functionality as it's been there since the very beginning and seems to have no purpose.